unstructured_inference/inference/layout.py

```diff
@@ -109,6 +153,10 @@ def __init__(
     def __str__(self):
         return "\n\n".join([str(element) for element in self.elements])
 
+    def to_string(self):
+        # Temporary method, should replace __str__
```

Comment:
This can be a doc string like `"""Temporary method, should replace __str__"""` instead of a comment with `#`